mobile/src/context/APIContext.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { createContext } from 'react';
+import { WordsAliveAPI } from '../classes/WordsAliveAPI';
+
+//set baseURL using environment varibales
+const baseURL = 'http://' + (process.env.BASE_URL || 'localhost:8080');
```

Comment:
The protocol should be part of the BASE_URL, since staging and prod are going to be over https. You can just do `const baseURL = process.env.BASE_URL || 'http://localhost:8080';`

Reply:
Also I thought we figured out we need to do this? https://docs.expo.io/guides/environment-variables/#using-app-manifest--extra What changed?

Reply:
Turns out for programs to have access to env variables, you need to export them which neither me or Aksharan were doing. If we use the babel config, we can just use process.env. No app.config needed

Reply:
The problem is:

> Unlike .env, this property is included when you publish your project with expo publish or expo build

When we build the app as a standalone APK/ISA this will all fall apart. If we reference environment variables in app.config.js, then:

> The manifest is loaded when starting or publishing your app and has access to the environment variables using process.env